artifacts/definitions/Windows/System/DLLs.yaml

```diff
@@ -11,7 +10,19 @@ parameters:
   - name: dllRegex
     description: A regex applied to the full dll path (e.g. whitelist all system dlls)
     default: .
-
+  - name: Calculate_Hash
+    default: Y
+    type: bool
+  - name: certificate
+    default: Y
+    type: bool    
+  - name: Type
+    type: choices
+    default: unsigned
+    choices:
+      - trusted
+      - unsigned
```

Comment:
I think we also need a choice to cover either

Reply:
```suggestion
      default: .
```